src/components/copy-tooltip/CopyTooltip.android.js

```diff
@@ -1 +1,51 @@
-export default () => null;
+import Clipboard from '@react-native-community/clipboard';
+import PropTypes from 'prop-types';
+import React, { useRef, useState } from 'react';
+import { TouchableOpacity, View } from 'react-native';
+import Tooltips from 'react-native-tooltips';
+import { colors } from '../../styles';
+
+const CopyTooltip = ({ children, textToCopy, tooltipText }) => {
+  const [visible, setVisible] = useState(false);
+  const parent = useRef();
+  const target = useRef();
+
+  return (
+    <>
```

Comment:
can you make this `<Fragment>` instead of `<>` 🙏️ we are trying to use `<Fragment>` consistently throughout the project